README.md

```diff
@@ -148,7 +148,7 @@ await fastify.register(import('@fastify/rate-limit'), {
 - `allowList`: array of string of ips to exclude from rate limiting. It can be a sync or async function with the signature `(request, key) => {}` where `request` is the Fastify request object and `key` is the value generated by the `keyGenerator`. If the function return a truthy value, the request will be excluded from the rate limit.
 - `redis`: by default, this plugin uses an in-memory store, but if an application runs on multiple servers, an external store will be needed. This plugin requires the use of [`ioredis`](https://github.com/redis/ioredis).<br> **Note:** the [default settings](https://github.com/redis/ioredis/blob/v4.16.0/API.md#new_Redis_new) of an ioredis instance are not optimal for rate limiting. We recommend customizing the `connectTimeout` and `maxRetriesPerRequest` parameters as shown in the [`example`](https://github.com/fastify/fastify-rate-limit/tree/master/example/example.js).
 - `nameSpace`: choose which prefix to use in the redis, default is 'fastify-rate-limit-'
-- `continueExceeding`: Renew user limitation when user sends a request to the server when still limited
+- `continueExceeding`: Renew user limitation when user sends a request to the server when still limited. This will take priority over exponentialBackoff
```

Comment:
```suggestion
- `continueExceeding`: Renew user limitation when user sends a request to the server when still limited. This will take priority over `exponentialBackoff`
```